spec/features/admin_edits_beer_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+feature 'Admin edits information for each individual beer page', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.